src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -252,12 +257,16 @@ function geometryToExtrudedPolygon(geometry, properties, options, multiGeomAttri
         const holesOffsets = polygonGeometry.holes.map(h => h.offset);
 
         // triangulate the top face
-        const start = vertices.length + polygonGeometry.contour.offset + offset;
+        let start = vertices.length + polygonGeometry.contour.offset + offset;
```

Comment:
Any idea why `end` doesn't depend on ` polygonGeometry.contour.offset`? (or why end isn't `start + count`)?

Reply:
I concede that I didn't think here, but it could make more sense (and more understandable).